src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/VersionCompatibility/DotNetCoreProjectCompatibilityDetector.cs

```diff
@@ -329,7 +329,9 @@ private static async Task<CompatibilityLevel> GetProjectCompatibilityAsync(Uncon
         {
             if (project.Capabilities.AppliesTo($"{ProjectCapability.DotNet} & {ProjectCapability.PackageReferences}"))
             {
-                IProjectProperties properties = project.Services.ActiveConfiguredProjectProvider.ActiveConfiguredProject.Services.ProjectPropertiesProvider.GetCommonProperties();
+                ConfiguredProject? activeConfiguredProject = project.Services.ActiveConfiguredProjectProvider.ActiveConfiguredProject;
+                Assumes.NotNull(activeConfiguredProject);
```

Comment:
Behaviour change. Substitutes NRE for whatever Assumes throws. `ActiveConfiguredProject` is nullable.